src/domain/session/room/timeline/deserialize.js

```diff
@@ -346,6 +346,10 @@ class Deserializer {
 }
 
 export function parseHTMLBody(platform, mediaRepository, allowReplies, html) {
+    if (allowReplies) {
```

Comment:
I think we might want to remove the flag entirely? If we want to still be able to fallback to the embedded replied-to text if we can't fetch the context event, I would add another function to give us the `MessageBody` for the embedded reply and parse **only** what is in the mx-reply tag.

Reply:
I've removed this flag but currently we don't rely on the fallback at all. If we can't fetch the context event, we render an error instead.